velox/functions/prestosql/window/AggregateWindow.cpp

```diff
@@ -217,12 +256,14 @@ class AggregateWindowFunction : public exec::WindowFunction {
     // the previous and current row.
     for (int i = 0; i < numRows; i++) {
       auto currentFrameEnd = rawFrameEnds[i] - startFrame + 1;
-      if (currentFrameEnd > prevFrameEnd) {
-        computeAggregate(rows, prevFrameEnd, currentFrameEnd);
+      if (rawFrameStarts[i] == -1 && rawFrameEnds[i] == -1) {
```

Comment:
Based on the incremental logic we should never have empty frames that satisfy those conditions right ? So do we need this check ?